.github/workflows/main.yml

```
@@ -4,14 +4,15 @@ on:
   push:
     branches: 
       - master
+      - windows
   pull_request:
 
 jobs:
   generate-test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - name: Set up Python 3.5.9
+      - name: Set up Python 3.5.5
```

```suggestion
      - name: Set up Python 3.5.9
```

なんかactionsがwindowsだと3.5.9が入らなかったので3.5.5にしてますが、ubuntuは3.5.9が入ってます

---

直しました